app.js

```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var app = express();
+
+// mongoose is just another node module
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/trucksAPI');
+var bodyParser = require('body-parser');
+
+var Truck = require('./models/truckModel');
+var idManager = require('./idManager');
+
+idManager.setIds(trucks.getTrucks());
+
+app.use(express.router('Routes', truckRoutes));
```

Comment:
The truckRoutes module is exporting middle-ware for you, so what you want to do here is to "mount" this middle-ware on a path. Before you do that, you'll want to require in the truckRoutes module first:

`var truckRouter = require('./routes/truckRoutes');

then mount that middle-ware on the '/trucks' path: 

`app.use('/trucks', truckRoutes);`